tests/cypress/integration/Home/commerceHomePage.cy.js

```diff
@@ -1,56 +1,310 @@
 import { GetPluginId } from '../wp-module-support/pluginID.cy';
 import { comingSoon } from '../wp-module-support/utils.cy';
+import { EventsAPI, APIList } from '../wp-module-support/eventsAPIs.cy';
 
 const customCommandTimeout = 60000;
+const pluginId = GetPluginId();
 
+describe( 'Commerce Home Page- Coming soon mode', () => {
+	before( () => {
+		cy.exec( `npx wp-env run cli wp plugin deactivate woocommerce`, {
+			failOnNonZeroExit: false,
+		} );
+		cy.visit( '/wp-admin/admin.php?page=' + pluginId + '#/home' );
+		comingSoon( true );
+	} );
 
```

Comment:
@sangeetha-nayak  We need to add checks for `Ready to go live` section & its CTAs 
<img width="1264" alt="Screenshot 2023-12-14 at 3 46 48 PM" src="https://github.com/newfold-labs/wp-module-ecommerce/assets/41899077/676a06b2-d844-4c09-a068-97d2d9602e0a">

Reply:
I will add, current slot contains tests on Live mode, Next steps when woo is inactive. These scenarios would be considered when woo is active.